recipe/meta.yaml

```diff
@@ -1,41 +1,57 @@
+{% set name = "types-jsonschema" %}
 {% set version = "4.19.0.4" %}
 
 package:
   name: types-jsonschema
   version: {{ version }}
 
+
 source:
-  - url: https://pypi.io/packages/source/t/types-jsonschema/types-jsonschema-{{ version }}.tar.gz
+  - url: https://pypi.io/packages/source/t/{{ name }}/{{ name }}-{{ version }}.tar.gz
     sha256: 994feb6632818259c4b5dbd733867824cb475029a6abc2c2b5201a2268b6e7d2
-  - url: https://raw.githubusercontent.com/python/typeshed/main/LICENSE
-    sha256: 295f8538c94ae5c3043301cf7cff1c852dab6a786a8ddee471e061b40d5ecabe
+
 
 build:
-  noarch: python
-  script: {{ PYTHON }} -m pip install . -vv --no-deps --no-build-isolation
   number: 0
+  script: {{ PYTHON }} -m pip install . -vv --no-deps --no-build-isolation
+  skip: true  # [py<36]
+
 
 requirements:
   host:
-    - python >=3.7
     - pip
+    - python
+    - wheel
+    - setuptools
   run:
-    - python >=3.7
+    - python
     - referencing
 
+
 test:
+  imports:
+    - types
+  requires:
+    - pip
   commands:
     - pip check
     - test -f $SP_DIR/jsonschema-stubs/__init__.pyi                 # [unix]
     - if not exist %SP_DIR%\\jsonschema-stubs\\__init__.pyi exit 1  # [win]
-  requires:
-    - pip
+
 
 about:
   home: https://github.com/python/typeshed
-  summary: Typing stubs for jsonschema
-  license: Apache-2.0
+  license: Apache-2.0 AND MIT
```

Comment:
The license provided on PyPI is Apache-2.0, see https://inspector.pypi.io/project/types-jsonschema/4.19.0.4/packages/01/1e/7bb26b307a3cd32da256ac6c5d60ccd8d01bb5e3c44e6c08bce680a765c5/types_jsonschema-4.19.0.4-py3-none-any.whl/types_jsonschema-4.19.0.4.dist-info/METADATA#line.6.

Reply:
I think PyPi is wrong. This is part of the [typeshed](https://github.com/python/typeshed) repo and that contains `Apache-2.0 AND MIT`.

`types-request` and `types-pyyaml` are also from `typeshed` and use this license.

Reply:
I am considering the source of truth to be the archive we download. If they make an error on the license in the PyPI metadata that's not our problem.